dlc-oracle-test/src/test/scala/org/bitcoins/dlc/oracle/DLCOracleTest.scala

```diff
@@ -81,6 +81,85 @@ class DLCOracleTest extends DLCOracleFixture {
     }
   }
 
+  it must "create a ranged event and get its details" in {
+    dlcOracle: DLCOracle =>
+      val time = TimeUtil.now
+      val eventName = "ranged"
+      val start = -100
+      val stop = 100
+      val step = 5
+
+      val expectedOutcomes = start.to(stop, step).map(_.toString)
```

Comment:
ahhhhhhhhh do we know how we feel about encodings for numeric events? I could imagine an argument that since they transport in TLVs as `int32` perhaps they should be serialized as such when hashing? Either way we need to be extra careful to make sure we match whatever spec is going to do on this

Reply:
Yeah agreed. This is mostly just waiting on the actual spec

Reply:
Now signs the int32's bytes in #2162